bump.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+const path = require("path");
+const { exec } = require("@actions/exec");
+
+(async () => {
+  await exec("pnpm changeset", ["status", "--output=./.changeset/status.json"]);
+  await exec("pnpm changeset", ["version"]);
```

Comment:
Why do we run changeset `twice`?

Reply:
The first changeset just generates the `status.json` that tells us which apps **will be** versioned. The second changeset is what does the actual versioning.
Or did you mean why we can't combine the 2?

Reply:
Combine the two if possible but should be bump be called **when** there is a changeset already?

Reply:
What I think happens is 
Locally, when you think the changes for an app are ready to be released, you run `changeset` which allows you to select the `app` that should be released. This will create a `.changeset` file like [.changeset/light-lions-hug.md](https://github.com/CodeForAfrica/ui/blob/30f577bd11826eb5138a7d2a38e47862b3a34c35/.changeset/light-lions-hug.md)  where you fully describe that changes that this release brings.
You then review the file and if everything looks good you create a PR that gets merged into main.
On push to main, the `release` workflow gets executed which should create a Changeset release PR that gets automatically updated anytime code is merged into main. We are then responsible for determining when the PR should be merged and thus causing the release to be deployed.
So to answer the question, bump is called when there is a changeset already inside the `.changeset` but there can be multiple changesets(multiple apps being released) in the folder and that's what running `status` changeset will help with.

Reply:
I see what you mean & it makes sense but:

If `changeset version` actually bumps the version in package.json **_and_** that's all we need, why not skip the whole status check? If for a given app there was no version change, there won't be any change to the corresponding `Dockerfile` i.e. we'll create the `Dockerfile` with the same version.

Reply:
Actually this was the first approach I had used but the issue is we won't have a way of knowing which app was actually changed unless we start doing git checks inside the workflow

Reply:
Why do we need to know this? The [`app-build.yml` will check](https://github.com/CodeForAfrica/ui/blob/b4e665ad6db1174c4446834762ec46f648bb39a3/.github/workflows/codeforafrica-build-deploy.yml#L11) if it's `package.json` has changed or not. Or are we dropping that approach?

My understanding is we need this script for one purpose only: to ensure for **all** apps, `contrib/dokku/Dockerfile` version matches the corresponding `package.json` version. The rest will be taken care of in the corresponding `app-build.yml`

Reply:
With this, it means we will have multiple release workflows for each app; Which I think is alright.